### PR TITLE
update tika version to 2.x

### PR DESCRIPTION
**Pull request by smcgrattan**

### CODE REVIEW

**Review by andrewbp — APPROVED** [review]

File: pom.xml [hunk 1]
```diff
@@ -116,12 +120,23 @@
         <groupId>com.fasterxml.jackson.module</groupId>
         <artifactId>jackson-module-jaxb-annotations</artifactId>
         <version>${dep.jackson.version}</version>
+        <exclusions>
```
Comment (fbruton) [inline pom.xml:123]: Suggest removing exclusion in favor of the following :

```
<dep.jakarta.activation.version>1.2.2</dep.jakarta.activation.version>
```
```
      <dependency>
        <groupId>jakarta.activation</groupId>
        <artifactId>jakarta.activation-api</artifactId>
        <version>${dep.jakarta.activation.version}</version>
      </dependency>
```

File: pom.xml [hunk 2]
```diff
@@ -38,9 +40,12 @@
     <dep.dropwizard.metrics.version>4.2.4</dep.dropwizard.metrics.version>
     <dep.errorprone.javac.version>9+181-r4173-1</dep.errorprone.javac.version>
     <dep.errorprone.version>2.10.0</dep.errorprone.version>
+    <dep.gson.version>2.8.6</dep.gson.version>
+    <dep.guava.version>30.1.1-jre</dep.guava.version>
```
Comment (fbruton) [inline pom.xml:44]:
```suggestion
```

**Review by fbruton — CHANGES_REQUESTED** [review]

**Review by fbruton — DISMISSED** [review]

File: pom.xml [hunk 3]
```diff
@@ -60,8 +65,7 @@
     <dep.slf4j.version>1.7.36</dep.slf4j.version>
     <dep.spullara.mustache.compiler.version>0.9.10</dep.spullara.mustache.compiler.version>
     <dep.spymemcached.version>2.12.3</dep.spymemcached.version>
-    <dep.tika.version>1.24</dep.tika.version>
-    <dep.versions.maven.plugin.version>2.14.2</dep.versions.maven.plugin.version>
+    <dep.tika.version>2.3.0</dep.tika.version>
```
Comment (fbruton) [inline pom.xml:68]:
```suggestion
    <dep.tika.version>2.3.0</dep.tika.version>
    <dep.versions.maven.plugin.version>2.14.2</dep.versions.maven.plugin.version>
```

**Review by fbruton — CHANGES_REQUESTED** [review]

**Comment by jpdahlke** [comment]
https://github.com/NationalSecurityAgency/emissary/pull/447

**Pull request closed without merging**